Assets/Scripts/SegmentDisplay/SegmentDigit.cs

```diff
@@ -0,0 +1,33 @@
+using UnityEngine;
+using UnityEngine.Assertions;
+
+public class SegmentDigit : MonoBehaviour
+{
+    public GameObject[] segments;
+    private int[][] digits = new int[][]{
```

Comment:
nit: Suggest making this static readonly, as it never changes

```suggestion
    private static readonly int[][] digits = new int[][]{
```